Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/eventregistration/attendee/confirm/ConfirmCheckInFragment.kt

```diff
@@ -24,7 +25,8 @@ class ConfirmCheckInFragment : Fragment(R.layout.fragment_confirm_check_in), Aut
         factoryProducer = { viewModelFactory },
         constructorCall = { factory, savedState ->
             factory as ConfirmCheckInViewModel.Factory
-            factory.create(navArgs.verifiedTraceLocation)
+            val editId = if (navArgs.editCheckInId == 0L) null else navArgs.editCheckInId
```

Comment:
should `editCheckInId ` be nullable ? if `0` represents a nullable value

Reply:
NavArguments didn't allow primitives like `long` to be `null`, the option was greyed out.